packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -399,7 +400,7 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
 
   return (
     <StyledBaseButton
-      ref={buttonRef as any}
+      ref={ref as never}
```

Comment:
Didn't get why the `never`?

Reply:
yeah any works too. not sure why i removed any. 

> tho quick tip: never is can be better in some cases because you don't need to ignore the eslint rule 😆